src/screens/HomeScreen.js

```diff
@@ -24,6 +24,130 @@ import { ScreenName } from '../types';
 
 const { MATOMO_TRACKING, ROOT_ROUTE_NAMES } = consts;
 
+const renderItem = ({ item }) => {
+  const {
+    buttonTitle,
+    categoriesNews,
+    fetchPolicy,
+    showData,
+    navigate,
+    navigation,
+    query,
+    queryVariables,
+    showVolunteerEvents,
+    title
+  } = item;
+
+  const NAVIGATION = {
+    CATEGORIES_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.CATEGORIES,
+        queryVariables: {},
+        rootRouteName: ROOT_ROUTE_NAMES.POINTS_OF_INTEREST_AND_TOURS
+      }
+    },
+    EVENT_RECORDS_INDEX: {
+      name: ScreenName.Index,
+      params: {
+        title,
+        query: QUERY_TYPES.EVENT_RECORDS,
+        queryVariables: { limit: 15, order: 'listDate_ASC' },
+        rootRouteName: ROOT_ROUTE_NAMES.EVENT_RECORDS
+      }
+    },
+    NEWS_ITEMS_INDEX: ({
+      categoryId,
+      categoryTitle,
+      categoryTitleDetail,
```

Comment:
we could rename them here directly

```
categoryTitle: title,
categoryTitleDetail: titleDetail,
```

in order to have a cleaner return with

```
params: {
  title,
  titleDetail,
  query: QUERY_TYPES.NEWS_ITEMS,
  queryVariables,
  rootRouteName
}
```

what do you think?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/786/commits/48dc731a6e9a172ff8def17024414d4f0145c493